winrm4j/src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -40,6 +41,10 @@ public WinRmToolResponse executeCommand(List<String> commands) {
     	return executeCommand(joinCommands(commands));
     }
 
+    public void setRequestTimeout(Integer requestTimeoutInSeconds) {
```

Comment:
I wonder if we should add a builder for `WinRmTool`. Or maybe mark this as `@Beta` for now?